openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -170,11 +170,9 @@ Credentials can therefore vary in the following dimensions:
 - Credential Format
 - Cryptographic Data
 
-In the context of a single request, all issued Credentials MUST share the same Credential Format and Credential Dataset. However, they MAY contain different Cryptographic Data, such as being bound to different cryptographic keys (e.g., to ensure unlinkability between the Credentials).
+In the context of a single request, all issued Credentials belonging to the batch MUST share the same Credential Format and Credential Dataset. However, they MAY contain different Cryptographic Data, such as being bound to different cryptographic keys (e.g., to ensure unlinkability between the Credentials).
 To issue multiple Verifiable Credentials with differing Credential Formats or Credential Datasets, multiple requests MUST be sent to the Credential Endpoint.
 
-In the course of the authorization process, the Credential Issuer MAY also request Credential presentation as a means to authenticate or identify the End-User during the issuance flow, as described in (#use-case-5).
```

Comment:
why was this deleted?

Reply:
@Sakurann I was editing the batch credential section. I didn't understand why a reference to presentation based authentication was here. I think it needs to be moved but maybe it makes more sense for me to restore it  and get this merged asap, and then it can be sorted out in https://github.com/openid/OpenID4VCI/pull/509

Reply:
I agree it seems to be placed in the wrong section. Can we move it somewhere more sensible than deleting it or is it already mentioned elsewhere?

Reply:
@paulbastian @Sakurann i moved the section to the introduction to authorization code flow.

Reply:
I'm happy with this change.